src/components/ContextMenu.tsx

```diff
@@ -52,14 +56,23 @@ const styles: SXStyles = {
   },
 };
 
-export const ContextMenu = ({ options, showDotDotDot }: ContextMenuType) => {
+export const ContextMenu = ({ options, showEllipsis }: ContextMenuType) => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
 
   const clickOption = (onClick: Function, args: any[] = []) => {
+    console.log(args)
+    console.log(onClick)
```

Comment:
Are the console logs here to stay or just temporary for debugging?

Reply:
nice catch. was a dev artifact